File: codes/ruby/008_operator_ternary.rb
print("\n", "# Ternary Operator in Ruby")

CORRECT_ANSWER = "foo"
print("\n", "correct answer: ", CORRECT_ANSWER)

my_answer = "bar"
print("\n", "my answer: ", my_answer)
print("\n", "is my answer correct: ", ((my_answer === CORRECT_ANSWER) ? "correct!" : "wrong!"))
# is my answer correct: wrong!

my_answer = "foo"
print("\n", "my answer: ", my_answer)
print("\n", "is my answer correct: ", ((my_answer === CORRECT_ANSWER) ? "correct!" : "wrong!"))
# is my answer correct: correct!

File: codes/python/008_operator_ternary.py
print("\n# Ternary Operator in Python")

CORRECT_ANSWER = 'foo'
print(f'correct answer: "{CORRECT_ANSWER}"')

my_answer = 'bar'
print(f'my answer: "{my_answer}"')
print(f'is my answer correct: {"correct!" if (my_answer == CORRECT_ANSWER) else "wrong!"}')
# is my answer correct: wrong!

my_answer = 'foo'
print(f'my answer: "{my_answer}"')
print(f'is my answer correct: {"correct!" if (my_answer == CORRECT_ANSWER) else "wrong!"}')
# is my answer correct: correct!
